### DISCUSSION TITLE
Request for Mobile TOC Optimization – Similar to Astro Docs

### DISCUSSION THREAD

**Discussion by xkfe** [lin-stephanie/astro-antfustyle-theme#6] [Ideas]
want to achieve a toc mobile experience similar to Astro Docs:

![image](https://github.com/user-attachments/assets/aef50295-390d-410f-be15-4864195d3cbd)

**Comment by lin-stephanie** [comment]
@xkfe Thanks for the TOC suggestion! There is a plan to enhance the TOC on mobile, but it is likely to follow the style of the existing navigation menu on mobile devices, instead of adopting the Astro Docs style. Would this be acceptable?

**Comment by xkfe** [comment]
@lin-stephanie Sure, looking forward to seeing it on mobile devices soon!

**Reply by lin-stephanie** [reply]
TOC support on mobile is live in commit c4d1137ac2f312655f3e6525f6bde1838b3b1568. Feel free to try it out!

**Reply by xkfe** [reply]
I received an email about the TOC update this morning and checked it out immediately. It’s great, but could the highlight for the current anchor be optimized a bit?

Also, I have a question: I want to create a Tags page where articles are assigned tags. How can I implement filtering articles by clicking on a tag? Since I’m not very familiar with Astro, I’d really appreciate any guidance!

**Reply by lin-stephanie** [reply]
Thanks for your message! I’m not sure what you mean about optimizing the highlight for the current anchor -- could you elaborate?

For creating a Tags page, the [Generate tag pages](https://docs.astro.build/en/tutorial/5-astro-api/2/) section in the Astro tutorial might be helpful. There is also a plan to add tag functionality to this theme.

**Reply by xkfe** [reply]
Such as highlight or selection effects for the current reading position, which makes it convenient to track where you are reading. 

![image](https://github.com/user-attachments/assets/67a1019a-69a6-41d7-9208-049d8a4fdba6)

**Reply by lin-stephanie** [reply]
Got it, I'll get on it ASAP.

**Reply by lin-stephanie** [reply]
@xkfe The feature your mentioned is now supported in https://github.com/lin-stephanie/astro-antfustyle-theme/commit/6eb536d4d46c040deace8c4e1c6ec9daa7844edb. Feel free to share your feedback.
